wordpress-mobile/WordPress-Android
# Issue/use material alert dialog

This PR switched a couple of remaining AlertDialog's to material versions.
Specifically, we switched from `AlertDialog.Builder()` to `MaterialAlertDialogBuilder()`

There are no substantial visual changes or changes in dialogs' behavior. 

To test:
There is no need to check all the updated dialogs, but you can check couple of easily accessible ones to confirm that they are working:
- Link dialog in Aztec.
- Delete site confirmation dialog.
- Country picker dialog in the domain registration flow.

PR submission checklist:
